cpp/src/slice2php/Main.cpp

```diff
@@ -703,27 +739,35 @@ CodeVisitor::visitClassDefStart(const ClassDefPtr& p)
             }
         }
     }
+}
 
-    endNamespace();
+bool
+CodeVisitor::visitExceptionStart(const ExceptionPtr& p)
+{
+    startNamespace(p);
 
-    if(_classHistory.count(scoped) == 0)
-    {
-        _classHistory.insert(scoped); // Avoid redundant declarations.
-    }
+    _out << sp << nl << "if (PHP_VERSION_ID < 70000)";
```

Comment:
Same here, keep it 8.2 only, no need to change the versions that were already working.